ROMFS/px4fmu_common/init.d/rc.interface

```diff
@@ -19,7 +19,7 @@ then
 
 	if [ $MIXER_AUX == none -a $USE_IO == yes ]
 	then
-		set MIXER_AUX ${MIXER}.aux
+		set MIXER_AUX ${MIXER}
```

Comment:
I'm pretty sure this was just wrong, as further down it is `set MIXER_AUX_FILE /etc/mixers/${MIXER_AUX}.aux.mix`. There is no file with `.aux.aux.mix`, so the only thing that can happen, is that we now select an aux file (as intented), whereas before we did not.
There are only a few aux mixer files:
```
./mixers/claire.aux.mix
./mixers/pass.aux.mix
./mixers/firefly6.aux.mix
./mixers/vtol_delta.aux.mix
./mixers/mount.aux.mix
./mixers/vtol_AAVVT.aux.mix
./mixers/vtol_AAERT.aux.mix
./mixers/mount_legs.aux.mix
```